app/scripts/controllers/form_add.js

```diff
@@ -185,4 +186,11 @@ angular.module('endslaverynowApp')
 				return collection
 			}
 
+			function prependHttp(url) {
+				if(/^(http)/.test(url)) {
```

Comment:
Do we want to include/worry about/use `https`?

Reply:
Also since you added testing in this PR, it would be good to have a test to show that it works :)

Reply:
Good question! It was hard to even find an http only website to test, so I think it'd be fine to use https. If you visit an http website using a url with https, then you get a security notice, but if you visit an https website using a url with http then it seems to redirect to the https site. I figured I'd default to http, since that always provides a link that works, and it gets switched to https before users would be doing anything with credit cards. Does that seem okay? It really was hard to find non-https sites so https is probably a safe default for retailers too.

Reply:
Haha and yes! My original intention for adding testing was to test this 🙈. Testing proved to be more challenging than I anticipated, but I'll give it another shot! It would be great to have it tested.